### PR TITLE
Added financial contributors to the README

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or another core contributor signed this repository up for Open Collective. This pull request adds financial contributors from your Open Collective https://opencollective.com/slimphp ❤️

  What it does:
  - adds a badge to show the latest number of financial contributors
  - adds a banner displaying contributors to the project on GitHub
  - adds a banner displaying all individuals contributing financially on Open Collective
  - adds a section displaying all organizations contributing financially on Open Collective, with their logo and a link to their website

P.S: As with any pull request, feel free to comment or suggest changes.

  Thank you for your great contribution to the Open Source community. You are awesome! 🙌
  And welcome to the Open Collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other Open Source sustainers!

### CODE REVIEW

**Comment by l0gicgate** [comment]
Thank you for doing this! This is great. I think we can remove the "Credits" section since the "Contributors" section takes care of it instead.

Also can we duplicate this PR for the Slim 4.x branch as well since we have an upcoming major release?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Slim Framework
 
 [![Build Status](https://travis-ci.org/slimphp/Slim.svg?branch=3.x)](https://travis-ci.org/slimphp/Slim)
-[![Coverage Status](https://coveralls.io/repos/github/slimphp/Slim/badge.svg?branch=3.x)](https://coveralls.io/github/slimphp/Slim?branch=3.x)
+[![Financial Contributors on Open Collective](https://opencollective.com/slimphp/all/badge.svg?label=financial+contributors)](https://opencollective.com/slimphp) [![Coverage Status](https://coveralls.io/repos/github/slimphp/Slim/badge.svg?branch=3.x)](https://coveralls.io/github/slimphp/Slim?branch=3.x)
```
Comment (adriansuter) [inline README.md]: I think we should bring the coverage status `[![Coverage Status]...` to a new line.

**Review by adriansuter — CHANGES_REQUESTED** [review]

**Comment by monkeywithacupcake** [comment]
I think I created a new pull on 4.x. I am not sure what you want to adjust in the credits. Please feel free to delete/edit as best works for your team.

**Comment by l0gicgate** [comment]
@monkeywithacupcake you pulled against the `3.x` branch as shown in the header of the pull request:


Also, it seems that the contributors svg is now broken. You can access it directly just fine however when you navigate to the [README](https://github.com/slimphp/Slim/blob/49ae771f71028c01aed3a88b08379eb6b60e18c1/README.md) on this pull it does not show anymore:
```
https://opencollective.com/slimphp/contributors.svg?width=890&button=false
```

I'm not going to merge until this is rectified from you guys' end

**Pull request merged**